### PR TITLE
update the comments in the examples

### PR DESCRIPTION
**Pull request by shavidissa**
- iterable-operations
-  passthrough
- websub-internal-hub-sample
- websub-remote-hub-sample
- jms-queue-message-producer-transactional
- jms-queue-message-producer
- jms-queue-message-receiver-sync-client-ack
- jms-queue-message-receiver-sync
- jms-queue-message-receiver-with-client-acknowledgment
- jms-queue-message-receiver
- jms-simple-durable-topic-message-subscriber
- jms-simple-queue-message-producer
- jms-simple-queue-message-receiver
- jms-simple-topic-message-producer
- jms-simple-topic-message-subscriber
- jms-topic-message-producer
- jms-topic-message-subscriber
- streams
- string-template
- streams-patterns
- streams-sequences
- streams-wthin-services

## Automation tests
 - N/A

## Security checks
N/A

## Samples
N/A

### CODE REVIEW

**Review by samgnaniah — COMMENTED** [review]
In iterable-operations.description, use the proper syntax of the terms, i.e., JSON and XML.

hub.bal is missing an article in line 1. Line 12, say "Wait for the subscriber to subscribe to this hub..." instead of "at".

**Review by samgnaniah — COMMENTED** [review]
Line 29 of jms-durable-topic-message-subscriber.bal makes me think this is an if statement. Can you reword.

File: examples/jms-durable-topic-message-subscriber/jms-durable-topic-message-subscriber.bal [hunk 1]
```diff
@@ -23,9 +23,10 @@ endpoint jms:DurableTopicSubscriber subscriber {
 // Bind the created subscriber to the listener service.
 service<jms:Consumer> jmsListener bind subscriber {
 
-    // OnMessage resource get invoked when a message is received.
+    // The `OnMessage` resource gets invoked when a message is received.
     onMessage(endpoint consumer, jms:Message message) {
-        // Retrieve the text message content with error lifting by using the check keyword.
+        // Retrieve the text message content using the `check` keyword. `Check` terminates the function if an error
```
Comment (uthaiyashankar) [inline examples/jms-durable-topic-message-subscriber/jms-durable-topic-message-subscriber.bal]: "Retrieve the text message content using the `check` keyword" is not correct. Check is just error lifting. Above statement means the text message is retrieved using the check keyword.

File: examples/jms-queue-message-receiver-sync-client-ack/jms-queue-message-receiver-sync-client-ack.bal [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
-// Initialize a Queue sender on top of the the created sessions
+// Initialize a Queue sender on top of the the created session.
 endpoint jms:QueueReceiver queueReceiver {
     session:jmsSession,
     queueName:"MyQueue"
 };
 
 public function main(string[] args) {
-    // Receive a message from the JMS provider.
+    // Waits for 1 seconds for the message to be received by the JMS provider.
```
Comment (uthaiyashankar) [inline examples/jms-queue-message-receiver-sync-client-ack/jms-queue-message-receiver-sync-client-ack.bal]: This should be wait for the message at-most 1 second. If the message is not received within 1 sec, then timeout should occur.

File: examples/jms-queue-message-receiver-sync-client-ack/jms-queue-message-receiver-sync-client-ack.bal [hunk 3]
```diff
@@ -1,36 +1,39 @@
...
     var result = queueReceiver -> receive(timeoutInMilliSeconds = 1000);
 
     match result {
         jms:Message msg => {
+            // If the message is received, this block is executed. It checks for client acknowledgement.
```
Comment (uthaiyashankar) [inline examples/jms-queue-message-receiver-sync-client-ack/jms-queue-message-receiver-sync-client-ack.bal]: It is "not" checking for client acknowledgement. It is just acknowledge the message.

File: examples/jms-queue-message-receiver-sync/jms-queue-message-receiver-sync.bal [hunk 4]
```diff
@@ -1,35 +1,38 @@
...
-// Initialize a queue receiver on top of the the created sessions
+// Initialize a queue receiver on top of the the created sessions.
 endpoint jms:QueueReceiver queueReceiver {
     session:jmsSession,
     queueName:"MyQueue"
 };
 
 public function main(string[] args) {
-    // Receive a message from the JMS provider.
+    // Waits for 5 seconds for the message to be received by the JMS provider.
```
Comment (uthaiyashankar) [inline examples/jms-queue-message-receiver-sync/jms-queue-message-receiver-sync.bal]: Same as above

File: examples/streams-join/streams-join.bal [hunk 5]
```diff
@@ -2,38 +2,38 @@ import ballerina/http;
 import ballerina/mime;
 import ballerina/io;
 
-
+// Create an object type named `ProductMaterial` and define the attributes.
```
Comment (uthaiyashankar) [inline examples/streams-join/streams-join.bal]: Shall we remove this comment? It states the obvious. If we are commenting, we should say why, not what.

File: examples/streams-join/streams-join.bal [hunk 6]
```diff
@@ -2,38 +2,38 @@ import ballerina/http;
...
 type ProductMaterial {
     string name;
     float amount;
 };
 
+// Create an object type named `MaterialUsage` and define the attributes.
```
Comment (uthaiyashankar) [inline examples/streams-join/streams-join.bal]: Shall we remove this comment? It states the obvious. If we are commenting, we should say why, not what.

**Review by uthaiyashankar — APPROVED** [review]

**Pull request merged**